skaffold.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: skaffold/v1beta15
```

Comment:
use `skaffold` to check running `docker build` successfully, when editing Dockerfile

Reply:
Add usage comment